css/src/components/banner.scss

```diff
@@ -0,0 +1,53 @@
+$banner-background: $info-background !default;
+$banner-padding: $spacer-5 !default;
+
+$banner-font-size: $font-size-7 !default;
+$banner-color: $info-dark !default;
+
+$banner-border-color: $border-white-high-contrast !default;
+$banner-border-width: $border-width !default;
+
+.banner {
+	@extend %layout-gap;
+
+	display: block;
+	position: relative;
+	outline-color: $text;
+	background-color: $banner-background;
+	color: $banner-color;
+	font-size: $banner-font-size;
+	padding-block: $banner-padding;
```

Comment:
Wondering if banners should also have some default inline padding.

Reply:
The layout-gap mixin is doing that I believe

Reply:
Yes, it is!